### PR TITLE
feat: upgrade config-godaddy parser option

### PR DESCRIPTION
**Pull request by mmason2-godaddy**

### CODE REVIEW

File: packages/eslint-config-godaddy/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
   plugins: ['mocha', 'json'],
   extends: ['eslint:recommended'],
   parserOptions: {
-    ecmaVersion: 9,
+    ecmaVersion: 13,
```
Comment (agerard-godaddy) [inline packages/eslint-config-godaddy/index.js:13]: Wow, this was pretty far behind.

**Review by agerard-godaddy — APPROVED** [review]

**Comment by mmason2-godaddy** [comment]
I can't merge just FYI

File: packages/eslint-config-godaddy/index.js (hunk 1 above)
Comment (agerard-godaddy) [reply packages/eslint-config-godaddy/index.js:13]: Actually, before we merge... is ecmaVersion 13 available on eslint v7? We may need to bump the peerDependencies and as a result require a new major release of this config

Comment (rmarkins-godaddy) [reply packages/eslint-config-godaddy/index.js:13]: @agerard-godaddy @mmason2-godaddy just as a follow up I am going to bump the `peerDependency` to v8 prior to release of a new major version

**Pull request merged**